scripts/variantstore/beta_docs/gvs-exome-workspace-description.md

```diff
@@ -0,0 +1,144 @@
+### Genomic Variant Store Exomes workflow (beta) for variant discovery
```

Comment:
Note - this document needs further additions / updates probably done by @kbergin